cmake/scripts/common/Macros.cmake

```diff
@@ -429,7 +429,9 @@ function(core_optional_dep)
       set(_required True)
     endif()
 
-    if(TARGET kodi::${dep})
+    if(TARGET kodi::${dep} OR
+       (${depup}_FOUND AND (${depspec} IN_LIST optional_buildtools OR
+                            ${depspec} IN_LIST required_buildtools)))
```

Comment:
Why check required_buildtools in core_optional_dep function here and also the lines below?

Reply:
changed. The next lines was more an abundance of caution, but have removed that as in theory the dep should not be in one of the required lists.